pkg/cmd/stack/destroy.go

```diff
@@ -175,6 +176,22 @@ func destroyServicesNotInStack(ctx context.Context, spinner *utils.Spinner, s *m
 		spinner.Start()
 	}
 
+	ingressesList, err := ingress.List(ctx, s.Namespace, s.GetLabelSelector(), c)
+	if err != nil {
+		return err
+	}
+	for i := range ingressesList {
+		if _, ok := s.Endpoints[ingressesList[i].Name]; ok {
+			continue
+		}
+		if err := ingress.Destroy(ctx, ingressesList[i].Name, ingressesList[i].Namespace, c); err != nil {
+			return fmt.Errorf("error destroying ingress '%s': %s", ingressesList[i].Name, err)
+		}
+		spinner.Stop()
```

Comment:
I think we should stop the spinner before returning the error to avoid weird UI behaviors.

Reply:
I think it's not necessary, when you return the error, the function that calls  `destroyServicesNotInStack`. [stops the spinner](https://github.com/okteto/okteto/blob/master/pkg/cmd/stack/destroy.go#L73)

It's the same way we have implemented destroying deployments and services that are not in the stack.